website/docs/language/state/remote-state-data.html.md

```diff
@@ -94,6 +95,38 @@ post-processing such as JSON decoding. You can then change that module later
 if you switch to a different strategy for sharing data between multiple
 Terraform configurations.
 
+## Example Usage Terraform Cloud
+
+This relies on the `tfe_outputs` data source in the [Terraform Cloud/Enterprise Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs).
+It is used to retrieve the state outputs for a given workspace.
+It enables output values in one Terraform configuration to be used in another.
+
+As the outputs retrieved from a different workspace may contain sensitive
+information in the context of that workspace, the `values` attribute of this
+data source is statically marked as
+[sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output).
+
+```hcl
+provider "tfe" {
+  hostname = "app.terraform.io"
+  token = "<token>"
+}
+
+data "tfe_outputs" "foo" {
+  organization = "my-org"
+  workspace = "my-workspace"
+}
+
+resource "random_vpc_resource" "vpc_id" {
+  keepers = {
+    # Generate a new ID any time the value of 'bar' in workspace 'my-org/my-workspace' changes.
+    bar = data.tfe_outputs.foo.values.bar
+  }
+
+  byte_length = 8
+}
+```
+
```

Comment:
Given this isn't example usage for the data source described on this page, I wonder if it's confusing to have it here? Maybe instead of this section, you should have a bold call-out in the `remote` backend example below, e.g:

> Note: When using Terraform Cloud/Enterprise, we recommend using [the `tfe_outputs` data source](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/data-sources/outputs) is recommended instead of `terraform_remote_state`.

Reply:
I debated having this example here, cause I thought that adding the header _Example Usage Terraform Cloud_ without an example is a bit off. But yeah, its not an example usage of the `terraform_remote_state`, so its probably better to just link to the data source. 

I'll update